code/game/objects/items/weapons/twohanded.dm

```diff
@@ -1127,3 +1128,107 @@
 			return ..()
 
 #undef BROOM_PUSH_LIMIT
+
+/obj/item/twohanded/supermatter  //Supermatter Halberd, used by Oblivion Enforcers
+	name = "supermatter halberd"
+	desc = "The revered weapon of Oblivion Enforcers, used to enforce the Order's will."
+	icon_state = "smhalberd0"
+	force = 5
+	sharp = TRUE
+	damtype = BURN
+	w_class = WEIGHT_CLASS_BULKY
+	slot_flags = SLOT_BACK
+	throwforce = 15
+	force_unwielded = 5
```

Comment:
Why is both `force` and `force_unwielded` necessary here?

Reply:
the other two-handed weapons in this file do it. I'm pretty sure 'force' is set to 'force_wielded' when you wield it but the default should be the same as the unwielded value